Adding PyGameZero Workshop (#699)
EuroPython/website: src/content/pages/pygamezero.mdx

---
title: PyGameZero Workshop

> subtitle: An introduction and fun workshop to teach beginners how to using Python to make games.

Reviewer: ```suggestion
subtitle: An introductionary and fun workshop to teach beginners how to use Python to create games.
```

---

# PyGameZero Workshop

> Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this 2-hour long workshop, we will learn about PyGameZero - a simple framework in Python that allow coding beginners to create a simple game using Python.

Reviewer: ```suggestion
Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this 2-hour long workshop, we will learn about PyGameZero - a simple framework in Python that allows coding beginners to create a simple game using Python.
```

> Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this short introduction workshop, we will learn about PyGameZero, a simple framework in Python that allow any coding beginners to create a simple game using Python.

Reviewer: ```suggestion
Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this short workshop, we will learn about PyGameZero - a simple framework in Python that allow coding beginners to create a simple game using Python.
```

Reviewer 2: Maybe add the duration of the workshop?

> In this **X hour-long** workshop, (...)

Or something on these lines :)

**When**: Wednesday 10th July 2024 10:45
**Where**: Open Space at EuroPython

> **Who**: EuroPython attendees (no extra registration required)

Reviewer: ```suggestion
**Who**: Anyone with a valid in-person EuroPython 2024 ticket - Conference or Combined (see [ticket types (https://ep2024.europython.eu/tickets#ticket-types) for details) - No extra registration required!
```

## FAQ

> - *I am a beginner in coding/ Python, can I join?*

Reviewer: ```suggestion
- *I am a beginner in coding / Python, can I join?*
```

> Sure! This workshop is for beginners in coding and Python, just bring your computer!

Reviewer: ```suggestion
Sure! This workshop is for beginners in coding and Python. Just bring your computer!
```

> - *I have kid(s) with me at the conference, can I bring my kid(s)?*

Reviewer: ```suggestion
- *I have kid(s) with me at the conference. Can I bring my kid(s)?*
```

> If your kid(s) is 7-12 years old and you will be accompanying them, yes, you are welcome to bring your kid(s).

Reviewer: ```suggestion
If your kid(s) is/are 7-12 years old  - and you will be accompanying them -, then yes, you are welcome to bring your kid(s).
```

- *What do I need to bring?*

> A charged computer that can connect to the internet with wifi, the creative energy and an open mind to learn.

Reviewer: ```suggestion
A charged computer that can connect to the internet with wifi, the creative energy, and an open mind to learn.
```

> A charged computer that can connect to the internet with wifi, the creative energy, and an open mind to learn.

Reviewer: ```suggestion
A charged computer that can connect to the internet with wifi, creative energy, and an open mind to learn.
```
